glusterd2/gdctx/global.go

```diff
@@ -73,6 +75,22 @@ func GenerateLocalAuthToken() error {
 			if errWrite := ioutil.WriteFile(authFile, []byte(LocalAuthToken), 0640); errWrite != nil {
 				return errWrite
 			}
+			usr, err := user.Lookup("gluster")
```

Comment:
Look for "gluster" group not "gluster" user.

Reply:
done